docs/pages/web/chartgraph.js

```diff
@@ -0,0 +1,599 @@
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Line"
+          description={`A line graph plots numeric values for categorical data as a line that shows a progression through time.
+
+Requires props:  \`type="line"\` and \`elements\` should be set to \`type="line"\`
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={line} name="Line" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Combo"
+          description={`This combines a bar graph with a line graph. It is useful to see both categories and a trend or range over time.
+
+Requires props:  \`type="combo"\` and \`elements\` should be set to \`type="line"\` \`type="bar"\`
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={combo} name="Combo" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Biaxial"
+          description={`Biaxial graphs show two y-axis. They're used when different amounts need to be displayed per each category represented in the graph.
+
+Requires props:  \`layout="verticalBiaxial"\` (default) or  \`layout="horizontalBiaxial"\`
+`}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={biaxial} name="Biaxial" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Title & description"
+          description={`A title for the graph in case it’s not displayed elsewhere on the screen. An optional description is available if more context is needed.
+
+Requires props:  \`title\` and \`description"\`.
+`}
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={title} name="Title & description" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Tooltip"
+          columns={2}
+          description={`For showing more precise details on hover.
+
+Requires props:  \`renderTooltip\`
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            title="Default tooltip"
+            sandpackExample={<SandpackExample code={tooltip} name="Default tooltip" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="lg"
+            title="No tooltip"
+            sandpackExample={<SandpackExample code={noTooltip} name="No tooltip" />}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Custom tooltip"
+            sandpackExample={<SandpackExample code={customTooltip} name="Custom tooltip" />}
```

Comment:
Custom tooltip looks similar to regular, am I missing something?

Maybe add a header or something, change the background, layout? 
<img width="351" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/f796d3bf-eba6-4dae-b2a9-156d434c98bd">



Default:
<img width="277" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/37ddc5c6-fe6b-4ea0-ba9a-95658084a6ca">

Custom:
<img width="217" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/c7972aed-b2a4-4549-89d0-f385ad5ef1ea">